gemini/sample-apps/e2e-gen-ai-app-starter-pack/README.md

```diff
@@ -163,7 +163,7 @@ See [deployment/README.md](deployment/README.md) for instructions.
 
    ```bash
    gcloud config set project YOUR_CI_CD_PROJECT_ID
-   gcloud services enable serviceusage.googleapis.com cloudresourcemanager.googleapis.com cloudbuild.googleapis.com secretmanager.googleapis.com
+   gcloud services enable serviceusage.googleapis.com cloudresourcemanager.googleapis.com cloudbuild.googleapis.com secretmanager.googleapis.com cloudscheduler.googleapis.com
```

Comment:
ideally we shouldn't change anything outside of the pattern folder. Including this, this api can be enabled by terraform

Reply:
good point, for now I've added the need for discoveryengine api in the readme in the right folder but clearing the cloud scheduler from here